src/input_formatter/text_formatter.test.tsx

```diff
@@ -91,10 +91,17 @@ test("replace detected expression-elements into readable prolog commands", () =>
 	expect(result).toEqual(expectedResult);
 });
 
+test("expression formatter testcase", () => {
+	emptyIssueList();
```

Comment:
You keep repeating `emptyIssueList()` in every test. The operations you repeat before every test can be perfectly be put into the set up. See the beforeEach function here: https://jestjs.io/docs/en/setup-teardown

Reply:
good idea. thx. i changed it